js_modules/dagster-ui/packages/ui-core/src/asset-graph/__fixtures__/AssetNode.fixtures.ts

```diff
@@ -646,6 +652,7 @@ export const LiveDataForNodePartitionedFresh: LiveDataForNodeWithStaleData = {
   lastMaterialization: {
```

Comment:
if we use `buildMaterializationEvent` to construct this then next time we add a field to this type the mocks will get it for free with no changes required.